scaling_performance/routing_optimization.adoc

```diff
@@ -22,32 +22,54 @@ xref:../install_config/router/index.adoc#install-config-router-overview[router]
 is the ingress point for all external traffic destined for {product-title}
 services.
 
-On an public cloud instance of size 4 vCPU/16GB RAM, a single HAProxy router is able to handle
-between 7000-32000 HTTP keep-alive requests depending on encryption, page size,
-and the number of connections used. For example, when using TLS
-xref:../architecture/networking/routes.adoc#edge-termination[edge] or
-xref:../architecture/networking/routes.adoc#re-encryption-termination[re-encryption]
-terminations with large page sizes and a high numbers of connections, expect to
-see results in the lower range. With HTTP keep-alive, a single HAProxy router is
-capable of saturating 1 Gbit NIC at page sizes as small as 8 kB.
-
-The table below shows HTTP keep-alive performance on such a public cloud
-instance with a single HAProxy and 100 routes:
-
-[cols="2,3,3",options="header"]
+When evaluating a single HAProxy router performance in terms of
+HTTP requests handled per second, the performance varies depending
+on many factors.  In particular:
+
+* HTTP keep-alive/close mode,
+
+* xref:../architecture/networking/routes.adoc#route-types[route type]
+
+* TLS session resumption client support
+
+* number of concurrent connections per target route
+
+* number of target routes 
+
+* backend server page size
+
+* underlying infrastructure (network/SDN solution, CPU...)
+
+While performance in your specific environment will vary, our lab
+tests on a public cloud instance of size 4 vCPU/16GB RAM, a single
+HAProxy router handling 100 routes terminated by backends serving
+1kB static pages is able to handle
```

Comment:
I would format this section a little differently.

.. 1KB static page is able to handle the following.

In HTTP keep-alive mode scenarios:
<table>

In HTTP close (no keep-alive) scenarios:
<table>